serverlessworkflow/modules/ROOT/pages/cloud/operator/build-and-deploy-workflows.adoc

```diff
@@ -0,0 +1,138 @@
+----
+
+[NOTE]
+====
+In this Custom Resource, `spec.platform.registry.secret` is the name of the secret you created just before.
+====
+
+You can also update "on-the-fly" the `KogitoServerlessPlatform` registry field with this command (change <YOUR_REGISTRY>)
+
+.Create a `KogitoServerlessPlatform` with a specific registry
+[source,bash,subs="attributes+"]
+----
+curl https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/{operator_version}/config/samples/sw.kogito_v1alpha08_kogitoserverlessplatform.yaml | sed "s|address: .*|address: <YOUR_REGISTRY>" | kubectl apply -f -
+----
+
+In order to retrieve the Cluster IP address of Minikube's internal registry to configure your platform, you can use the following command:
+
+.Retrieve Minikube registry internal IP
+[source,bash,subs="attributes+"]
+----
+kubectl get svc registry -n kube-system -ojsonpath='{.spec.clusterIP}'
+----
+
+== Build and deploy your workflow application
+
+You can now send your workflow definition (`KogitoServerlessWorkflow`) to the operator.
+
+You can find a basic `KogitoServerlessWorkflow` in the link:https://github.com/kiegroup/kogito-serverless-operator/tree/{operator_version}/config/samples[`config/samples` folder] that is defining the link:{kogito_greeting_example_url}[{product_name} Greeting example].
+
+[source,bash,subs="attributes+"]
+----
+kubectl apply -f https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/{operator_version}/config/samples/sw.kogito_v1alpha08_kogitoserverlessworkflow.yaml -n kogito-workflows
+----
+You can check the logs of the build of your workflow via:
+
+.Get the workflow application pod logs
+[source,bash,subs="attributes+"]
+----
+kubectl logs kogito-greeting-builder -n kogito-workflows
+----
+
+The final pushed image must be printed into the logs at the end of the build.
+
+== Check the workflow application is running
+
+In order to check that the {product_name} Greeting application is up and running, you can try to perform a test HTTP call, from the greeting pod.
+
+.Check the greeting application is running
+[source,bash,subs="attributes+"]
+----
+kubectl patch svc greeting -n kogito-workflows -p '{"spec": {"type": "NodePort"}}'
+GREETING_SVC=$(minikube service greeting -n kogito-workflows --url)
+curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d '{"name": "John", "language": "English"}' $GREETING_SVC/greeting
+----
+
+If everything is working well you should receive a response like this:
+
+.Response from the greeting application
+[source,json,subs="attributes+"]
+----
+{"id":"b5fbfaa3-b125-4e6c-9311-fe5a3577efdd","workflowdata":{"name":"John","language":"English","greeting":"Hello from JSON Workflow, "}}
+----
+
+// TODO: add a troubleshooting guide - https://issues.redhat.com/browse/KOGITO-8864
```

Comment:
isn't that done ?

Reply:
@radtriste JIRA at least still open, @davidesalerno could you confirm this is still pending?

Reply:
yes